### DISCUSSION TITLE
PGNet训练时到eval时报Not supported format.错误

### DISCUSSION THREAD

**Discussion by Stonesusu** [PaddlePaddle/PaddleOCR#12976] [Q&A]
![image](https://github.com/PaddlePaddle/PaddleOCR/assets/29848394/71742b64-e185-4ae9-9b03-59055ed555aa)
训练100steps后进行eval，报如上错误

**Comment by SWHL** [comment]
从所给信息看不出来啥，需要你那里多跑几次eval_mode看看
